js/blocks/MeterBlocks.js

```diff
@@ -1026,6 +1037,29 @@ function setupMeterBlocks() {
             if (arg1 <= 0) logo.noteValuePerBeat[turtle] = 4;
             else logo.noteValuePerBeat[turtle] = 1 / arg1;
 
+            // setup default strong / weak beats until any strong beat block is used 
+
+            if (logo.noteValuePerBeat[turtle] == 4 && logo.beatsPerMeasure[turtle] == 4) {
+                logo.beatList[turtle].push(1);
+                logo.beatList[turtle].push(3);
+                logo.defaultStrongBeats[turtle] = true;
+            }
+            else if (logo.noteValuePerBeat[turtle] == 4 && logo.beatsPerMeasure[turtle] == 2) {
+                logo.beatList[turtle].push(1);
+                logo.defaultStrongBeats[turtle] = true;
+            }
+            else if (logo.noteValuePerBeat[turtle] == 4 && logo.beatsPerMeasure[turtle] == 3) {
+                logo.beatList[turtle].push(1);
+                logo.defaultStrongBeats[turtle] = true;
+            }
+            else if (logo.noteValuePerBeat[turtle] == 8 && logo.beatsPerMeasure[turtle] == 6) {
+                logo.beatList[turtle].push(1);
+                logo.beatList[turtle].push(4);
+                logo.defaultStrongBeats[turtle] = true;
+            }
```

Comment:
Something in this particular code snippet is interfering with our project.

Reply:
I will look into it

Reply:
Thanks. It might help to compare the previous behavior of the project to the new one. I've attached it above. Might have to do something with multiple action blocks, but I'm not sure.

Reply:
since this commit assigns strong beats by default , when you use a 4:4 meter in the action (transition) - (1, 3) become strong beats... 
this is a issue we have been working on , we cant use strong beats and every note do together #1898 , #2258 
for now you can remove the meter in the action (transition)

Reply:
Oh, I see. Yes, those issues have been bothersome from time to time. Happy to do any testing.